wordcloud/wordcloud.py

```diff
@@ -203,6 +208,23 @@ def __init__(self, font_path=None, width=400, height=200, margin=2,
         if font_path is None:
             font_path = FONT_PATH
         self.font_path = font_path
+        ttf = TTFont(
+            font_path, 0, verbose=0, allowVID=0,
+            ignoreDecompileErrors=True,
+            fontNumber=-1
+        )
+        supported_chars = []
+
+        for table in ttf["cmap"].tables:
```

Comment:
I wouldn't put this in the init, but rather in the building of the word cloud. Also, it would be great to make this somehow optional so as not to get too many dependencies. It would also be great to factorize this functionality somewhat. Init and process text are already too long.